ci/common

```diff
@@ -55,14 +55,20 @@ setup_kind () {
 setup_helm () {
     echo "setup helm ${Z2JH_HELM_VERSION}"
     if ! [ -f "bin/helm-${Z2JH_HELM_VERSION}" ]; then
-        curl -sSLo "bin/helm-${Z2JH_HELM_VERSION}.tar.gz" "https://storage.googleapis.com/kubernetes-helm/helm-v${Z2JH_HELM_VERSION}-linux-amd64.tar.gz"
+        curl -sSLo "bin/helm-${Z2JH_HELM_VERSION}.tar.gz" "https://get.helm.sh/helm-v${Z2JH_HELM_VERSION}-linux-amd64.tar.gz"
         tar -xf "bin/helm-${Z2JH_HELM_VERSION}.tar.gz" --directory bin --strip-components 1 linux-amd64/helm
         rm "bin/helm-${Z2JH_HELM_VERSION}.tar.gz"
         mv bin/helm "bin/helm-${Z2JH_HELM_VERSION}"
     fi
     cp bin/helm-${Z2JH_HELM_VERSION} bin/helm
 }
 
+init_helm () {
+    if [ "${Z2JH_HELM_VERSION%%.*}" = "2" ]; then
```

Comment:
```suggestion
    if [ "${Z2JH_HELM_VERSION:0:1}" = "2" ]; then
```

I googled `%%` to learn what it meant. It was "remove suffix" matching what follows. And matching everything removed everything except the first character. Aha... But if we want to only show the first character, I figured it made sense to use this logic instead as that may be quicker to understand for someone like me for example that isn't a linux user since birth ;D

Reply:
`%%.*` means remove everything starting from the first `.` that's found, so it'll still work if helm reaches a double digit major version 😀. I'm happy to go with your suggestion though.